mlflow/entities/metric.py

```diff
@@ -73,6 +84,11 @@ def dataset_digest(self) -> Optional[str]:
         """String. Digest of the dataset associated with the metric."""
         return self._dataset_digest
 
+    @property
+    def experiment_id(self) -> Optional[str]:
+        """String. Experiment ID associated with the metric."""
+        return self._experiment_id
```

Comment:
I don't think we need to expose the experiment ID on the entity either, especially if it's difficult for us to resolve it for preexisting metrics entries from runs. Let's remove it.

Reply:
Makes sense :)